src/cr/cube/matrix/measure.py

```diff
@@ -56,18 +56,36 @@ def means(self):
         """_Means measure object for this cube-result"""
         return _Means(self._dimensions, self, self._cube_measures)
 
+    def pairwise_means_indices(self, alpha, only_larger):
+        """_PairwiseMeansIndices measure object for this cube-result"""
+        return _PairwiseMeansIndices(
+            self._dimensions, self, self._cube_measures, alpha, only_larger
+        )
+
     def pairwise_p_vals_for_subvar(self, subvar_idx):
-        """_PairwiseSigPVals measure object for this cube-result and selected subvar."""
+        """_PairwiseSigPVals measure object for this cube-result and selected subvar"""
         return _PairwiseSigPVals(
             self._dimensions, self, self._cube_measures, subvar_idx
         )
 
     def pairwise_t_stats_for_subvar(self, subvar_idx):
-        """_PairwiseSigTStats measure object for this cube-result and selected subvar."""
+        """_PairwiseSigTStats measure object for this cube-result and selected subvar"""
         return _PairwiseSigTStats(
             self._dimensions, self, self._cube_measures, subvar_idx
         )
 
+    def pairwise_significance_means_p_vals(self, column_idx):
+        """_PairwiseMeansSigPVals measure object for this cube-result."""
```

Comment:
Docstring should explain what `column_idx` is. Also, we're using `column_idx` in some places, and `subvar_idx` in others. Is this the `subvar_idx` or not? It's not clear on the first read (without going back to the description of the PR).